tasks/main.yml

```diff
@@ -1,33 +1,45 @@
 ---
 
-- include_tasks: set-parse-version.yml
+- name: Set Facts for Datadog Agent Major Version
+  include_tasks: set-parse-version.yml
+
+- name: Debian Install Tasks
+  include_tasks: pkg-debian.yml
 
-- include_tasks: pkg-debian.yml
   when: ansible_os_family == "Debian"
 
-- include_tasks: pkg-redhat.yml
+- name: RedHat Install Tasks
+  include_tasks: pkg-redhat.yml
   when: ansible_os_family == "RedHat"
 
-- include_tasks: pkg-suse.yml
+- name: Suse Install Tasks
+  include_tasks: pkg-suse.yml
   when: ansible_os_family == "Suse"
 
-- include_tasks: pkg-windows.yml
+- name: Windows Install Tasks
+  include_tasks: pkg-windows.yml
   when: ansible_os_family == "Windows"
 
-- include_tasks: agent5-linux.yml
-  when: datadog_agent_major_version|int == 5 and ansible_os_family != "Windows"
+- name: Agent5 Install Tasks (Linux)
+  include_tasks: agent5-linux.yml
+  when: datadog_agent5 and ansible_os_family != "Windows"
 
-- include_tasks: agent-linux.yml
-  when: datadog_agent_major_version|int > 5 and ansible_os_family != "Windows"
+- name: Agent5 Install Tasks (Linux)
+  include_tasks: agent6-linux.yml
```

Comment:
```suggestion
  include_tasks: agent-linux.yml
```

Reply:
@the-real-cphillips I think you forgot or misclicked on this one, the main (non-Agent 5) linux install file is named [agent-linux.yml](https://github.com/the-real-cphillips/ansible-datadog/blob/name_parameter_for_includes/tasks/agent-linux.yml) now

Reply:
you are correct 🙈 
I just fixed in latest commit